build.sbt

```diff
@@ -21,6 +21,7 @@ lazy val cassandra = (project in file("cassandra"))
                        .settings(mySettings:_*)
                        .settings(name := "filodb-cassandra")
                        .settings(libraryDependencies ++= cassDeps)
+                       .settings(assemblySettings:_*)
```

Comment:
I was unable to complete build without adding this line. We can check during our meeting.